lib/tasks/server/express-server.js

```diff
@@ -16,7 +16,9 @@ class ExpressServerTask extends Task {
     this.emitter = new EventEmitter();
     this.express = this.express || require('express');
     this.http = this.http || require('http');
-    this.https = this.https || require('https');
+    // This should be migrated to require('http2') when we have migrated to express@5 which supports the built-in http2 bindings
```

Comment:
Why can't we require express@5?

Reply:
express@5 is still in beta, do we want to use the beta?

Reply:
express@5 has been in alpha for years (still at alpha.8, published 9 months ago)

Reply:
ugh, ok, that is a tad frustrating 😝